WcaOnRails/app/models/group.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Group < ApplicationRecord
+  enum group_type: {
```

Comment:
Super nit-pick, but I thought I'd mention it: The Rails 6 way of doing enums is:

```ruby
enum my_column, {
  foo: "foo",
  bar: "bar",
}
```

Note the **comma** in `enum :my_column, {}` as opposed to `enum my_column: {}`. In our code, we use the latter notation a lot because it was created before Rails 6. But now at least with new features we might as well stick to the new notation.